storage/client_range_test.go

```diff
@@ -146,7 +146,7 @@ func TestRejectFutureCommand(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	if v := val.GetInteger(); v != 15 {
+	if v := val.MustGetInteger(); v != 15 {
```

Comment:
You've got a good eye. I'll rename it to `mustGetInteger` and place it in one of the test files.